docs/stdlib/range.rst

```diff
@@ -105,61 +105,61 @@ cast into a :eql:type:`json` structure:
 
 .. code-block:: edgeql-repl
 
-    db> select <json>range(1, 10);
+    edgedb> select <json>range(1, 10);
     {"inc_lower": true, "inc_upper": false, "lower": 1, "upper": 10}
 
 It's also possible to cast in the other direction - from :eql:type:`json` to a
 specific range type:
 
 .. code-block:: edgeql-repl
 
-    db> select <range<int64>>to_json('{
-    ...   "lower": 1,
-    ...   "inc_lower": true,
-    ...   "upper": 10,
-    ...   "inc_upper": false
-    ... }');
+    edgedb> select <range<int64>>to_json('{
```

Comment:
Style guide note: db> and ... are used for edgeql-repl blocks (keeps output as short as possible which is nice for mobile)